osu.Game/Overlays/Options/General/LoginOptions.cs

```diff
@@ -10,38 +11,72 @@
 
 namespace osu.Game.Overlays.Options.General
 {
-    public class LoginOptions : OptionsSubsection
+    public class LoginOptions : OptionsSubsection, IOnlineComponent
```

Comment:
What if we provided some kind of harness to furnish the API with that implemented this interface, but in a private subclass of this UI component?